lib/assessment/grade.py

```diff
@@ -170,7 +170,7 @@ def tsv_data_from_choices(self, info, rubric, student_id):
                     tsv_data_choices.append(tsv_data)
 
         if len(tsv_data_choices) == 0:
-            raise "No valid responses. An InvalidResponseError should have been raised earlier."
+            return None
```

Comment:
Yes to the 2nd case. This was one of the tests that was failing, and after looking through the code, the error message seemed redundant, since invalid responses would be caught before getting to this point.

Reply:
Should it raise instead, then?